File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Regions/IRegionData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Regions
{
    public interface IRegionData :  CoreObject.IObjectID, CoreObject.IObjectTitle, CoreObject.IObjectName, CoreObject.IObjectOrder, CoreObject.IObjectDescription, CoreObject.IObjectState, IRegionParent
    {

    }
}

File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Regions/IRegionData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Regions;


use  Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface IRegionData extends IRegionParent,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{

}
